nav2_map_server/src/map_server/main_2d.cpp

```diff
@@ -13,18 +13,21 @@
 // limitations under the License.
 
 #include <memory>
-#include <stdexcept>
 #include <string>
 
 #include "nav2_map_server/map_server.hpp"
+#include "nav_msgs/msg/occupancy_grid.hpp"
+
 #include "rclcpp/rclcpp.hpp"
 
 int main(int argc, char ** argv)
 {
   std::string node_name("map_server");
 
   rclcpp::init(argc, argv);
-  auto node = std::make_shared<nav2_map_server::MapServer>();
+  auto node = std::make_shared<nav2_map_server::MapServer<nav_msgs::msg::OccupancyGrid>>();
```

Comment:
Semantically, I don't really see why this shouldn't be `MapServer2D` without template arguments using the map server in the 2d directory named as such with the `template <>` removed. The actual object didn't make use of the template argument so this seems like TMI and more confusing to a later reader.